File: StaffManagementAPI/Models/Staff.cs
using System.ComponentModel.DataAnnotations;

namespace StaffManagementAPI.Models
{
    public class Staff : BaseObject
    {
        [StringLength(8)]
        public string StaffID { get; set; }
        [StringLength(100)]
        public string FullName { get; set; }
        public DateTime Birthday { get; set; }
        public GenderType Gender { get; set; }
    }

    public enum GenderType
    {
        Male = 1,
        Female = 2
    }
}

File: staff-management-app/src/app/models/staff/staff.ts

export class Staff {
    id: number;
    staffID: string;
    fullName: string;
    birthday: string;
    genders: string;
    gender: number;
    createdDateTime: Date

    constructor(id: number, staffID: string, fullName: string, birthday: string, genders: string, gender: number, createdDateTime: Date) {
        this.id = id;
        this.staffID = staffID;
        this.fullName = fullName;
        this.birthday = birthday;
        this.genders = genders;
        this.gender = gender;
        this.createdDateTime = createdDateTime
    }
}
